### PR TITLE
Removed bad fallback from default config

### PR DESCRIPTION
**Pull request by oxtoacart**

### CODE REVIEW

**Comment by oxtoacart** [comment]
@fffw If you have time and feel comfortable, would you mind looking at this?

**Comment by myleshorton** [comment]
Lgtm

**Comment by oxtoacart** [comment]
Thanks for the review @myleshorton !

File: src/github.com/getlantern/flashlight/genconfig/lantern.yaml [hunk 1]
```diff
@@ -5,14 +5,6 @@ client:
...
-      authtoken: "Dhidw0Bb3AuxVoGumctTDFnMPolyfyEq4qo05l3VQosVO759UtUlwlFKeGV13cIH"
-      pipelined: true
-      weight: 1000000
```
Comment (maartje1) [inline src/github.com/getlantern/flashlight/genconfig/lantern.yaml]: Dhidw0Bb3AuxVoGumctTDFnMPolyfyEq4qo05l3VQosVO759UtUlwlFKeGV13cIH

**Pull request merged**